get-started/index.md

```diff
@@ -56,127 +56,156 @@ redirect_from:
+Each **virtual machine** (VM) runs a "guest" operating system (OS) with only
+virtual access to host resources through a hypervisor. VMs are resource
+intensive, and the resulting disk image and application state is an entanglement
+of OS settings, system-installed dependencies, OS security patches, and other
+easy-to-lose, hard-to-replicate ephemera. VMs provide an environment with far
+more resources than most applications need.
 
-### Virtual Machine diagram
+By contrast, a **container** runs applications natively and shares the kernel of
+the host machine with other containers, keeping it lightweight. Each container
+runs in a discrete process, taking no more memory than any other executable. The
+only information needed in a container image is the executable and its package
+dependencies, which never need to be installed on the host system. Because
+containers house their dependencies, a containerized app is portable and “runs
+anywhere.”
 
-![Virtual machine stack example](https://www.docker.com/sites/default/files/VM%402x.png)
+![Container stack example](https://www.docker.com/sites/default/files/Container%402x.png){:width="300px"} | ![Virtual machine stack example](https://www.docker.com/sites/default/files/VM%402x.png){:width="300px"}
 
-Virtual machines run guest operating systems&#8212;note the OS layer in each
-box. This is resource intensive, and the resulting disk image and application
-state is an entanglement of OS settings, system-installed dependencies, OS
-security patches, and other easy-to-lose, hard-to-replicate ephemera.
+## Prepare your Docker environment
 
-### Container diagram
+Install a [supported version](https://docs.docker.com/engine/installation/#supported-platforms){:target="_blank"}
+of Docker Community Edition (CE) or Enterprise Edition (EE).
 
-![Container stack example](https://www.docker.com/sites/default/files/Container%402x.png)
+For full [Kubernetes integration on Docker for Mac](https://docs.docker.com/docker-for-mac/kubernetes/){:target="_blank"},
+install [17.12.0-ce Edge](https://docs.docker.com/docker-for-mac/release-notes/#edge-release-notes){:target="_blank"} or higher.
 
-Containers can share a single kernel, and the only information that needs to be
-in a container image is the executable and its package dependencies, which never
-need to be installed on the host system. These processes run like native
-processes, and you can manage them individually by running commands like `docker
-ps`&#8212;just like you would run `ps` on Linux to see active processes.
-Finally, because they contain all their dependencies, there is no configuration
-entanglement; a containerized app "runs anywhere."
+[Install Docker](/engine/installation/index.md){: class="button outline-btn"}
+<div style="clear:left"></div>
 
-## Setup
+### Test Docker version
 
-Before we get started, make sure your system has the latest version of Docker
-installed.
+Ensure that you have a supported version of Docker:
```

Comment:
"supported" isn't the right word here. Maybe "recent" or "maintained"?

Reply:
Thanks. I meant supported platform per: https://docs.docker.com/engine/installation/#supported-platforms